index.html

```diff
@@ -0,0 +1,131 @@
+<html>
+<head>
+    <title>Resume Page</title>
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link rel="stylesheet" href="style.css" type="text/css"/>
+   
+</head>
+<body>
+    <div class="flex-container">
+
+        <div class="flex1">
```

Comment:
Naming should be meaningful, this comment applies throughout

Reply:
Done